Dockerfile

```diff
@@ -1,11 +1,9 @@
-FROM elixir:1.8-alpine
+FROM alpine:3.8
```

Comment:
A heads up on using Alpine images vs Ubuntu: https://superuser.com/questions/1219609/why-is-the-alpine-docker-image-over-50-slower-than-the-ubuntu-image